### DISCUSSION TITLE
Size of projects

### DISCUSSION THREAD

**Discussion by timurturbil** [community/community#11764] [Other Feature Feedback, Questions, & Ideas] [answered]
To Whom It May Concern,

Please show the size of projects published on github. There are a lot of people who want to know the size of a project before clone or download the project. 

Regards

**Comment by ramazanpolat** [comment]
I was about to submit a feature request exacly for this. Because a repo I've start downloading was looking like a 20-30 MB repo until it reached 500+ MB, and still downloading...

This is so fundamental, I can't understand why it is not implemented yet.

**Comment by jsoref** [comment, marked as answer]
Fwiw, this information is available:

https://api.github.com/repos/torvalds/linux
  "size": 3672815,

https://api.github.com/repos/homebrew/brew
  "size": 57783,

https://api.github.com/repos/github/.github
  "size": 18,

**Reply by timurturbil** [reply]
Thank you for your answer. There is already an answer on github. I missed it. 
https://stackoverflow.com/questions/8646517/how-can-i-see-the-size-of-a-github-repository-before-cloning-it

**Reply by jsoref** [reply]
The above as a given, it'd be nice if one didn't have to URL hack and then try to decipher JSON and then guess the units of a field in order to figure out how large a repository is before deciding whether they're interested in cloning it.
